internal/provider/models/log_streams_models.go

```diff
@@ -43,3 +44,10 @@ type SplunkHttpEventCollectorModel struct {
 	HecSourceName       types.String `tfsdk:"hec_source_name"`
 	Tls                 types.Bool   `tfsdk:"tls"`
 }
+
+type CrowdstrikeHttpEventCollectorModel struct {
+	HecHostPort   types.String `tfsdk:"hec_host_port"`
+	APIKey        types.String `tfsdk:"api_key"`
+	HecSourceName types.String `tfsdk:"hec_source_name"`
```

Comment:
```suggestion
	HecSourceName types.String `tfsdk:"source_name"`
```
Please implement this fix on the Splunk model as well